Utilities/itkSurfaceImageCurvature.h

```diff
@@ -260,7 +259,6 @@ class SurfaceImageCurvature :
   NeighborhoodIteratorType m_ti2;
   bool                     m_UseLabel;
   float                    m_kSign;
-  float                    m_Sigma;
```

Comment:
m_Sigma is already in the base class.  This shadow variable with the same name was confusing.  A minor side effect is that m_Sigma is of type "RealType" as defined in the base class.